diff --git a/dist/client.js b/dist/client.js
index d1d60dba8dbc35bd8ddde1166b5a308de93279a6..fb41aea70589394f8c56ad149a24567ecf1397ca 100644
--- a/dist/client.js
+++ b/dist/client.js
@@ -8,7 +8,7 @@ exports.getMDXExport = getMDXExport;
 
 var React = _interopRequireWildcard(require("react"));
 
-var _jsx_runtime = _interopRequireWildcard(require("react/jsx-runtime.js"));
+var _jsx_runtime = _interopRequireWildcard(require("react/jsx-runtime"));
 
 var ReactDOM = _interopRequireWildcard(require("react-dom"));
 
