diff --git a/node_modules/esbuild-register/dist/node.js b/node_modules/esbuild-register/dist/node.js
index a348a20..08ed43d 100644
--- a/node_modules/esbuild-register/dist/node.js
+++ b/node_modules/esbuild-register/dist/node.js
@@ -2703,7 +2703,7 @@ function register(esbuildOptions = {}) {
     const format = inferPackageFormat(dir, filename);
     const {code: js, warnings, map: jsSourceMap} = _esbuild.transformSync.call(void 0, code, {
       sourcefile: filename,
-      sourcemap: "both",
+      sourcemap: false,
       loader: getLoader(filename),
       target: options.target,
       jsxFactory: options.jsxFactory,
