diff --git a/lib/main.js b/lib/main.js
index 488cc1aea9802b3d6ae13aee27556403bec55d1c..263d68a1520dd9402ef4231d30bc74c3aba5c821 100644
--- a/lib/main.js
+++ b/lib/main.js
@@ -1,7 +1,7 @@
 var QRCode = require('./../vendor/QRCode'),
     QRErrorCorrectLevel = require('./../vendor/QRCode/QRErrorCorrectLevel'),
-    black = "\033[40m  \033[0m",
-    white = "\033[47m  \033[0m",
+    black = "\x1b[40m  \x1b[0m",
+    white = "\x1b[47m  \x1b[0m",
     toCell = function (isBlack) {
         return isBlack ? black : white;
     },
