diff --git a/ios/Sources/Extensions+Helpers/RCTView+Helpers.swift b/ios/Sources/Extensions+Helpers/RCTView+Helpers.swift
index 09be306d5aa39337c5114c2ad6ba7513218e0751..9a7f1166d0aa94f76cecd82b511ddec329a4a242 100644
--- a/ios/Sources/Extensions+Helpers/RCTView+Helpers.swift
+++ b/ios/Sources/Extensions+Helpers/RCTView+Helpers.swift
@@ -25,17 +25,6 @@ public extension RCTView {
     return rootView.recursivelyFindSubview(whereType: targetType);
   };
   
-  var closestParentReactContentView: RCTRootContentView? {
-    let targetType = RCTRootContentView.self;
-  
-    if let match = self.recursivelyFindParentView(whereType: targetType) {
-      return match;
-    };
-    
-    guard let rootView = self.rootViewForCurrentWindow else { return nil };
-    return rootView.recursivelyFindSubview(whereType: targetType);
-  };
-  
   var reactTouchHandlers: [RCTTouchHandler]? {
     self.gestureRecognizers?.compactMap {
       $0 as? RCTTouchHandler
@@ -58,6 +47,6 @@ public extension RCTView {
       currentView = superview;
     };
     
-    return closestParentReactContentView?.reactTouchHandlers?.first;
+    return nil;
   };
 };
