diff --git a/lib/module/ReanimatedModule/js-reanimated/webUtils.web.js b/lib/module/ReanimatedModule/js-reanimated/webUtils.web.js
--- a/lib/module/ReanimatedModule/js-reanimated/webUtils.web.js
+++ b/lib/module/ReanimatedModule/js-reanimated/webUtils.web.js
@@ -1,30 +1,3 @@
-'use strict';
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let createReactDOMStyle;
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let createTransformValue;
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let createTextShadowValue;
-try {
-  createReactDOMStyle =
-  // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports, n/no-unpublished-require
-  require('react-native-web/dist/exports/StyleSheet/compiler/createReactDOMStyle').default;
-  // eslint-disable-next-line no-empty
-} catch (_e) {}
-try {
-  // React Native Web 0.19+
-  createTransformValue =
-  // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports, n/no-unpublished-require
-  require('react-native-web/dist/exports/StyleSheet/preprocess').createTransformValue;
-  // eslint-disable-next-line no-empty
-} catch (_e) {}
-try {
-  createTextShadowValue =
-  // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports, n/no-unpublished-require
-  require('react-native-web/dist/exports/StyleSheet/preprocess').createTextShadowValue;
-  // eslint-disable-next-line no-empty
-} catch (_e) {}
-//# sourceMappingURL=webUtils.web.js.map
+'use strict';
+export { default as createReactDOMStyle } from 'react-native-web/dist/exports/StyleSheet/compiler/createReactDOMStyle.js';
+export { createTextShadowValue, createTransformValue } from 'react-native-web/dist/exports/StyleSheet/preprocess.js';
