diff --git a/lib/module/ReanimatedModule/js-reanimated/webUtils.web.js b/lib/module/ReanimatedModule/js-reanimated/webUtils.web.js
index 742885f31bbe3f3d84741dfb6c82156d40725c9f..84563c1b058926096b925738a3d69b4b6af8d9c7 100644
--- a/lib/module/ReanimatedModule/js-reanimated/webUtils.web.js
+++ b/lib/module/ReanimatedModule/js-reanimated/webUtils.web.js
@@ -1,30 +1,7 @@
 'use strict';
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let createReactDOMStyle;
+import createReactDOMStyle from 'react-native-web/dist/exports/StyleSheet/compiler/createReactDOMStyle';
+import { createTextShadowValue, createTransformValue } from 'react-native-web/dist/exports/StyleSheet/preprocess';
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let createTransformValue;
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export let createTextShadowValue;
-try {
-  createReactDOMStyle =
-  // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports, n/no-unpublished-require
-  require('react-native-web/dist/exports/StyleSheet/compiler/createReactDOMStyle').default;
-  // eslint-disable-next-line no-empty
-} catch (_e) {}
-try {
-  // React Native Web 0.19+
-  createTransformValue =
-  // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports, n/no-unpublished-require
-  require('react-native-web/dist/exports/StyleSheet/preprocess').createTransformValue;
-  // eslint-disable-next-line no-empty
-} catch (_e) {}
-try {
-  createTextShadowValue =
-  // eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports, n/no-unpublished-require
-  require('react-native-web/dist/exports/StyleSheet/preprocess').createTextShadowValue;
-  // eslint-disable-next-line no-empty
-} catch (_e) {}
+export { createReactDOMStyle, createTextShadowValue, createTransformValue };
 //# sourceMappingURL=webUtils.web.js.map
