diff --git a/dist/index.js b/dist/index.js
index 534bc959746bc9499c4f797b59bd804cc0dc3841..bcab24c0527fa2b5e5d9df66fb3c81531bec076f 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -9,9 +9,10 @@ var __createBinding = (this && this.__createBinding) || (Object.create ? (functi
 var __exportStar = (this && this.__exportStar) || function(m, exports) {
     for (var p in m) if (p !== "default" && !Object.prototype.hasOwnProperty.call(exports, p)) __createBinding(exports, m, p);
 };
+const interopRequire = x => x.default ? x : { default: x }
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.ArgumentError = exports.Predicate = void 0;
-const callsites_1 = require("callsites");
+const callsites_1 = interopRequire(require("callsites"));
 const infer_label_1 = require("./utils/infer-label");
 const predicate_1 = require("./predicates/predicate");
 Object.defineProperty(exports, "Predicate", { enumerable: true, get: function () { return predicate_1.Predicate; } });
