diff --git a/node_modules/react-native-web/dist/exports/Text/index.js b/node_modules/react-native-web/dist/exports/Text/index.js
index e4a6a93..027ddc6 100644
--- a/node_modules/react-native-web/dist/exports/Text/index.js
+++ b/node_modules/react-native-web/dist/exports/Text/index.js
@@ -27,6 +27,7 @@ var forwardPropsList = {
   accessible: true,
   children: true,
   classList: true,
+  className: true,
   dir: true,
   importantForAccessibility: true,
   lang: true,
@@ -152,13 +153,13 @@ var Text = forwardRef(function (props, forwardedRef) {
 Text.displayName = 'Text';
 var classes = css.create({
   text: {
-    border: '0 solid black',
-    boxSizing: 'border-box',
+    // border: '0 solid black',
+    // boxSizing: 'border-box',
     color: 'black',
     display: 'inline',
-    font: '14px System',
-    margin: 0,
-    padding: 0,
+    // font: '14px System',
+    // margin: 0,
+    // padding: 0,
     whiteSpace: 'pre-wrap',
     wordWrap: 'break-word'
   },
diff --git a/node_modules/react-native-web/dist/exports/TextInput/index.js b/node_modules/react-native-web/dist/exports/TextInput/index.js
index 57519b1..6aa2ff3 100644
--- a/node_modules/react-native-web/dist/exports/TextInput/index.js
+++ b/node_modules/react-native-web/dist/exports/TextInput/index.js
@@ -377,7 +377,7 @@ var TextInput = forwardRef(function (props, forwardedRef) {
   supportedProps.classList = classList; // 'auto' by default allows browsers to infer writing direction
 
   supportedProps.dir = dir !== undefined ? dir : 'auto';
-  supportedProps.enterkeyhint = returnKeyType;
+  supportedProps.enterKeyHint = returnKeyType;
   supportedProps.onBlur = handleBlur;
   supportedProps.onChange = handleChange;
   supportedProps.onFocus = handleFocus;
diff --git a/node_modules/react-native-web/dist/exports/View/index.js b/node_modules/react-native-web/dist/exports/View/index.js
index 54bb2e3..6cda3f6 100644
--- a/node_modules/react-native-web/dist/exports/View/index.js
+++ b/node_modules/react-native-web/dist/exports/View/index.js
@@ -26,6 +26,7 @@ var forwardPropsList = {
   accessibilityValue: true,
   accessible: true,
   children: true,
+  className: true,
   classList: true,
   disabled: true,
   importantForAccessibility: true,
diff --git a/node_modules/react-native-web/dist/modules/createDOMProps/index.js b/node_modules/react-native-web/dist/modules/createDOMProps/index.js
index 211c00a..1d2488a 100644
--- a/node_modules/react-native-web/dist/modules/createDOMProps/index.js
+++ b/node_modules/react-native-web/dist/modules/createDOMProps/index.js
@@ -177,6 +177,10 @@ var createDOMProps = function createDOMProps(component, props) {
     domProps.className = className;
   }
 
+  if (props.className) {
+    domProps.className = `${domProps.className || ''} ${props.className}`
+  }
+
   if (style) {
     domProps.style = style;
   } // OTHER
diff --git a/node_modules/react-native-web/dist/modules/normalizeColor/index.js b/node_modules/react-native-web/dist/modules/normalizeColor/index.js
index 223451f..65366d3 100644
--- a/node_modules/react-native-web/dist/modules/normalizeColor/index.js
+++ b/node_modules/react-native-web/dist/modules/normalizeColor/index.js
@@ -16,8 +16,13 @@ var normalizeColor = function normalizeColor(color, opacity) {
 
   if (color == null) return;
 
-  if (typeof color === 'string' && isWebColor(color)) {
-    return color;
+  if (typeof color === 'string') {
+    if (isWebColor(color)) {
+      return color;
+    }
+    if (color[0] === '-') {
+      return color
+    }
   }
 
   var colorInt = processColor(color);
